### ISSUE TITLE
Bug when running migrations without specifying connection

### ISSUE THREAD

**Issue by girardinsamuel** [MasoniteFramework/orm#288]
Requirements:
- masoniteorm==0.9.3
- masonite == develop

To reproduce: 
- in config/databases.py => set default as 'postgres'.
- run `python craft migrate` => 
```
AttributeError
  'str' object has no attribute 'get'
  at ~/.pyenv/versions/pingcrm_masonite/lib/python3.9/site-packages/masoniteorm/schema/Schema.py:120 in get_connection_information
      116│         return self._blueprint
      117│ 
      118│     def get_connection_information(self):
      119│         return {
    → 120│             "host": self.connection_details.get(self.connection, {}).get("host"),
      121│             "database": self.connection_details.get(self.connection, {}).get(
      122│                 "database"
      123│             ),
      124│             "user": self.connection_details.get(self.connection, {}).get("user"),
```

It's working when running `python craft migrate -c postgres`.

Here is the issue is that self.connection equals to `default` and then `self.connection_details.get(self.connection, {})` equals to the string `postgres`.

Is this a real bug or did I use wrong versions ? 🤔

**Comment by girardinsamuel** [comment]
Looks like it's working with unreleased version of masonite orm.
I installed in my project
```
pip install https://github.com/MasoniteFramework/orm
```
this version is not released and fixes this bug.

**Comment by girardinsamuel** [comment]
@josephmancuso FYI

**Comment by josephmancuso** [comment]
Released v0.9.4 which had a bunch of fixes we've been working on
